README.md

```diff
@@ -146,17 +161,66 @@ Note that you'll need to pass the configuration file each time you run `mix` wit
 
 **NOTE** If you're using persistent `geth` and `geth` is restarted after the above step, the authority account must be unlocked again:
 
-```
+```bash
 geth attach http://127.0.0.1:8545
 personal.unlockAccount(“<authority_addr from ~/config.exs>”, 'ThisIsATestnetPassphrase', 0)
 ```
 The passphrase mentioned above originates from [`dev_helpers`](apps/omg_eth/test/support/dev_helpers.ex).
 It is what is used when deploying the contract in the `dev` environment using `prepare_env!()` as above.
 
+##### Manual contract deployment
+
+Alternatively, the contract can be deployed manually. This is especially helpful when running on a `geth` instance 
+with limited `Ether` supply. From `elixir-omg` root directory:
+```bash
+mix compile
+iex -S mix run --no-start
+``` 
+       
+And then in `IEx`
+```elixir
+alias OMG.Eth.DevHelpers
+alias OMG.Eth
+
+import Eth.Encoding
+
+{:ok, _} = Application.ensure_all_started(:ethereumex)
+
+{:ok, accounts} = Ethereumex.HttpClient.eth_accounts()
```

Comment:
`{:ok, [faucet | _]} = ...` looks better and is more robust, but still has a ton of assumptions.

Regardless of the finer-grained comments, I think this script duplicates `prepeare_env!` a lot, while it could be made a rather straightforward extension of that. I think it should take less default assumptions and take some arguments (like the `faucet` address, that is only assumed unlocked and funded, instead of assuming it is the first acc known to `geth`).

Then, in this readme here, you just explain the usage of the modified `prepare_env!()` script + make a note about the necessity of funding the faucet (of course, this is sth like "personal faucet", that the user maintains, uses and reuses for (possibly) other testnet needs.

Lastly, mention that the `authority` account needs refunding from time to time (but this should rather be done using `geth attach` or `MEW` or sth